src/deepsparse/pipeline.py

```diff
@@ -193,6 +203,17 @@ def __call__(self, *args, **kwargs) -> BaseModel:
 
         return pipeline_outputs
 
+    def submit(self, *args, **kwargs) -> concurrent.futures:
+        """
+        Asynchronous inference, returns a future
+        """
+        if not self._threadpool:
+            raise ValueError(
+                "Cannot submit jobs to an uninitialized threadpool"
```

Comment:
This message suggests that there is a thread pool and that it needs to be initialized. But if I understand correctly, there is no thread pool at all, so perhaps the error should reflect that.

Reply:
Instead of raising the error, we piggyback on sequential execution now